src/core/agent/modules/signify/signifyApi.ts

```diff
@@ -74,13 +88,18 @@ export class SignifyApi {
     );
     try {
       await this.signifyClient.connect();
+      this.isOnline = true;
```

Comment:
This should be set after the try catch block... on first load (new signify bran) it will go into the catch block to call the boot endpoint first